src/authentication-service.ts

```diff
@@ -445,7 +445,7 @@ export class RedHatAuthenticationService {
       scope: scope,
       account: {
         id: claims.sub,
-        label: claims.preferred_username || claims.email || 'user@example.com',
+        label: claims.preferred_username || claims.email || 'email not found',
```

Comment:
```suggestion
        label: claims.preferred_username ?? claims.email ?? 'email not found',
```

Reply:
In the cases where `claim.preferred_username === ''` and/or `claim.email === ''` page would not show anything in place of username. It would show 'email not found' only if both claims do not defined in the token.

For me '' equals 'email not found'.